maquetado/setup/scss/titles.scss

```diff
@@ -0,0 +1,16 @@
+.title {
+  display: flex;
+  padding: 0px 0px 0px 50px;
+  align-items: center;
+}
+
+.title img {
```

Comment:
I do not recommend using html elements to apply styles (`img`), its always better to add a `class="something"`